app/src/main/java/com/shagalalab/constitution/di/Injections.kt

```diff
@@ -23,8 +24,12 @@ val dataModule = module {
     single { get<ConstitutionDatabase>().partDao() }
 }
 
+val module = module {
+    single { Executors.newSingleThreadExecutor() }
```

Comment:
As I have mentioned in the issue itself, let's explicitly set the return type to `Executor`, not the default `ExecutorService`